scene/gui/spin_box.cpp

```diff
@@ -255,6 +256,21 @@ bool SpinBox::is_editable() const {
 	return line_edit->is_editable();
 }
 
+void SpinBox::set_arrow_step(double p_arrow_step) {
+
+	arrow_step = p_arrow_step;
+}
+
+double SpinBox::get_arrow_step() const {
+
+	return arrow_step;
+}
+
+double SpinBox::_get_correct_step() {
```

Comment:
"correct" is probably a bad name here. You mean there is also an incorrect step? And maybe step ethics? :smiley: 

Instead, this might be inlined or renamed to `_get_arrow_step` / `_get_computed_arrow_step` / `_get_button_step`.

Reply:
Step ethics is a very complicated and political matter. We need to be very careful around it. ;)

Inlining it or _get_computed_arrow_step sounds like the way to go. I'll look into it.

Thank you very much

Reply:
Ended up inlining it